index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Richard Mbuti</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style></style>
+  </head>
+  <body>
+    <header class="topHeader">
+      <a href="#" class="logo">RK</a>
+      <nav class="menu">
+        <a href="#" class="toggleNav">
+          <img src="img/Union.png" alt="menu_button" />
```

Comment:
:red_circle: I think this hamburger icon image you are trying to display doesn't exist. Kindly make sure it is part of the project files.
 
Your Design 
![notthere](https://user-images.githubusercontent.com/26736582/205442478-412c69b7-fc58-4237-a7a4-cc1bcd87f51d.png)